spec/controllers/idv/sessions_controller_spec.rb

```diff
@@ -3,6 +3,8 @@
 describe Idv::SessionsController do
   render_views
 
+  let(:provider) { 'https://rp1.serviceprovider.com/auth/saml/metadata' }
+  let(:authn_context) { Saml::Idp::Constants::LOA2_AUTHNCONTEXT_CLASSREF }
```

Comment:
You are not wrong AFAIK. I just re-used LOA2 since it was already used elsewhere in the tests.